appscale/tools/remote_helper.py

```diff
@@ -225,6 +225,28 @@ def spawn_nodes_in_cloud(cls, options, count=1):
     """
     agent = InfrastructureAgentFactory.create_agent(options.infrastructure)
     params = agent.get_params_from_args(options)
+
+    # If we have running instances under the current keyname, we try to
+    # re-attach to them. If we have issue finding the location file or the
```

Comment:
You chose the branch to work off of... :)
We all own the code and the responsibility that comes with it.